packages/views/accounts_header_buttons.jade

```diff
@@ -1,11 +1,18 @@
 template(name="accountsHeaderButtons")
   if currentUser
-    li
-      a(href="{{path route='profileEdit'}}") Profile
-    li.account
-      a.sign-out.navbar-btn(href="#")
-        i.fa.fa-sign-out
-        span Sign Out
+    li.dropdown
+      a.dropdown-toggle.admin-settings(href='#' data-toggle='dropdown' aria-expanded='false')
+        span Account
+        i.fa.fa-gear
```

Comment:
Both I guess - for example people might not be used to sign out being in something like that.